src/components/tool_tip/tool_tip.tsx

```diff
@@ -61,6 +61,11 @@ interface ToolTipStyles {
   opacity?: number;
 }
 
+const displayToClassNameMap = {
+  inlineBlock: undefined,
+  block: 'eui-displayBlock',
```

Comment:
I'm going to blame @snide for this one 😆 

Yes, this will most likely do what they need, but not how we should be applying prop-based class names. We shouldn't be using our CSS utilities here. Instead, we'll want to, and by convention usually do, create a new class and apply the appropriate styling in the `tool_tip.scss` file.

```suggestion
  block: 'euiToolTipAnchor--displayBlock',
```

Reply:
The reason I didn't do it here is because it creates extra lines of compiled css. For the one in popover for example, it comes with a bunch of extra stuff. For this one though it ONLY applies the block.

I'm not hung up on it, and will change it, but that was my reasoning.

Reply:
Hmm. That's actually an interesting thought. I certainly see pros and cons to both methods. I think its best to stick with our conventions for now over the slight increase in CSS. Thanks for changing it.